wikipedia-sentences/README.md

```diff
@@ -66,59 +92,114 @@ shuf input.txt > input.txt
 
 To shuffle a file on macos, please read [this post](https://apple.stackexchange.com/questions/142860/install-shuf-on-os-x/195387).
 
-## 🔍 Search
+## Step 3. 🔍 Search your data
+Jina offers several different ways to search (query) your data. In this example, we show three of the most common ones. All three are optional, in a production environment, you would only choose one which suits your use case best. 
+
 
-### With REST API
+### Using a REST API
+Begin by running the following command to open the REST API interface.
 
 ```sh
 python app.py -t query_restful
 ```
 
-Then:
+You should open another terminal window and paste the following command. 
+
+```sh
+curl --request POST -d '{"top_k": 5, "mode": "search",  "data": ["hello world"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:45678/search'
+```
+
+Once you run this command, you should see a JSON output returned to you. This contains the five most semantically similar Wikipedia sentences to the text input you provided in the 'data' parameter. Feel free to alter the text in the 'data' parameter and play around with other queries! For a better understanding of the parameters see the table below. 
+|  |  |
+|--|--|
+| `top_k` | Integer determining the number of sentences to return |
+| `mode` | Mode to trigger in the call. See [here](https://docs.jina.ai/chapters/rest/) for more details |
+| `data` | Text input to query |
+ 
+
+
+
+
+### Using JinaBox; our frontend search interface
+
+**JinaBox** is a light-weight, highly customizable JavaScript based front-end search interface. To use it for this example, begin by opening the REST API interface. 
```

Comment:
I think we're calling it "Jina Box" in most places?

Reply:
Done

Reply:
Done